lib/android.dart

```diff
@@ -325,8 +331,11 @@ void _applyLaunchBackgroundXml({
 
   if (showBranding && gravity != brandingGravityValue) {
     //add branding when splash image and branding image are not at the same position
+    final androidRandingItemXml = _androidBrandingItemXml.replaceAll(
```

Comment:
`androidRandingItemXml` is spelled wrong here.

Reply:
I've fixed the typo.
Please check again @jonbhanson 

Thanks